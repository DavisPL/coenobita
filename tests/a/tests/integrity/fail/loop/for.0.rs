#![allow(unused_variables)]

fn main() {
    #[cnbt::tag({bin}{bin})]
    let mut x = 5;

    for i in 0..c::value() {
        x += 6;
    }
}