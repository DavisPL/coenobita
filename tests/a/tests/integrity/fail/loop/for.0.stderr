error: expected {bin}{bin}, found {bin}{bin,c}
 --> tests/integrity/fail/loop/for.0.rs:8:14
  |
8 |         x += 6;
  |              ^
